content/docs/custom-plugins/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Installing Custom Plugins
+lastUpdated: ''
```

Comment:
@martina-if The build will pass if you put an ISO formatted timestamp in here. e.g. `2021-03-16T21:00:00.0Z`

The time portion will be discarded but the date portion is used to display the doc was "published" on. Like this:

![Screenshot 2021-03-16 at 16 21 07](https://user-images.githubusercontent.com/562403/111343748-a4d95680-8673-11eb-9737-c83f4163f6a0.png)

Ideally it would say "Last updated at [date]" instead of "published" on but it's hopefully good enough for now.

Reply:
Thanks! fixed